LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerHttpClient.cpp

```diff
@@ -90,6 +90,10 @@ void ULootLockerServerHttpClient::SendRequest_Internal(HTTPRequest InRequest) co
 			else {
 				response.Error = response.ErrorData.Message;
 			}
+            FString RetryAfterHeader = Response->GetHeader("retry-after");
+            if(!RetryAfterHeader.IsEmpty()) {
+                response.ErrorData.Retry_after_seconds = FCString::Atoi(*RetryAfterHeader);
+            }
```

Comment:
Indentation, or at least I think so, so many curly brackets to keep track of 😛

Reply:
Fixed